WorkPackages/applications.tex

```diff
@@ -83,8 +83,8 @@
 \begin{wpdeliv}[due=1,miles=startup,id=opendose-analysis,dissem=PU,nature=DEM,lead=INSERM]
   {Jupyter services for nuclear medicine dosimetry with the OpenDose project}
 \end{wpdeliv}
-\begin{wpdeliv}[due=1,miles=startup,id=application-astro,dissem=PU,nature=R,lead=CDS]
-  {Report on development of Jupyter-based framework to access, visualize and analyze astronomical data available at CDS}
+\begin{wpdeliv}[due=1,miles=startup,id=application-astro,dissem=PU,nature=DEM,lead=CDS]
+    {Demonstrator of Jupyter notebooks providing access to astronomical data in CDS services including capabilities for analysis of catalogue data and interactive visualisation}
```

Comment:
Is this demonstrator accessible from EOSC? That would be ideal.

Second best might be to have interactive examples (Voila or conventional Binder) than make use of our Binder instance - in which case these demonstrator tools would also be available on the EOSC. Would be nice to mention the EOSC link as we try to provide innovative services for EOSC.